builtin/logical/pkiext/pkiext_binary/acme_test.go

```diff
@@ -292,3 +318,170 @@ func Test_ACMERSAPSSHybrid(t *testing.T) {
 
 	RunACMERootTest(t, "rsa", 2048, true, "ec", 256, false)
 }
+
+// Test_ACMEIPSans verify that we can perform ACME validations on IP and DNS identifiers using the Golang ACME library.
+func Test_ACMEIPSans(t *testing.T) {
+	cluster, vaultNetwork, _, vaultNodeID, vaultBindedPort := setupVaultDocker(t)
+	defer cluster.Cleanup()
+
+	setupAcme(t, cluster, "127.0.0.1", vaultBindedPort.HostPort, "rsa", 2048, true, "ec", 256, false)
```

Comment:
Ah, I was going to say, I figured we could add this to the existing test but then I realized that these ports wouldn't work out... Hmmm :D 

I'm fine with this though.

Reply:
I'm actually a little surprised this worked in CI the first time...